### DISCUSSION TITLE
(solved) Error while compilation of last bêta release

### DISCUSSION THREAD

**Discussion by bodinjm** [JGMaker3dofficial/artistd#32] [Q&A]
Hello i did a compilation of the last release (I just modified the menu language from en to fr using the VS environment. It went ok but with a warning/error saying "adc_... declared as extern" do you have that ? Thanks in advance

**Comment by mikejr83** [comment]
Are you getting the success message?

```
Environment    Status    Duration
-------------  --------  ------------
mks_robin_pro  SUCCESS   00:00:40.640
```

Have you set `LCD_LANGUAGE` like this:

```c++
#define LCD_LANGUAGE fr
```

I'm compiling with no issues from _release/2021.2_. I'll check from the tag commit.

**Reply by bodinjm** [reply]
yes

**Reply by bodinjm** [reply]
the message is .platformio/packages/framework-arduinoststm32-maple@2.10000.200103/STM32F1/libraries/STM32ADC/src/utility/util_adc.c:10:30: warning: 'adc_result' initialized and declared 'extern'
 extern volatile unsigned int adc_result = 0;

**Reply by bodinjm** [reply]
I have a Successful file but 1 error in the compilation

**Reply by mikejr83** [reply]
That's interesting!

Attached is the firmware compiled for French LCD language.

[Robin_pro.zip](https://github.com/JGMaker3dofficial/artistd/files/6038077/Robin_pro.zip)

Let me know if it works.

**Reply by bodinjm** [reply]
Thanks a lot

**Reply by bodinjm** [reply]
I will check that after diner ;)

**Comment by bodinjm** [comment]
Solved! I made a spelling mistake in a copy and paste
